src/screens/stationSelect.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Text, Button } from 'react-native-paper';
+import DropDownComponent from '../components/dropDown';
+import { ScreenNavigationProps } from '../routes';
+import { DropdownItem } from '../dropDownTypes';
+import { urlOpenerFunc } from '../helpers/urlOpener';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+  },
+  dropdownContainer: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+    margin: 10,
+  },
+  dropdown: {
+    margin: -10,
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type StationSelectProps = ScreenNavigationProps<'StationSelect'>;
+
+const stList: DropdownItem[] = [
+  {
+    label: 'Kings Cross',
+    value: 'KGX',
+  },
+  {
+    label: 'Euston',
+    value: 'EUS',
+  },
+  {
+    label: 'Canley',
+    value: 'CNL',
+  },
+  {
+    label: 'Newcastle',
+    value: 'NCL',
+  },
+  {
+    label: 'Orpington',
+    value: 'ORP',
+  },
+];
+
+const StationSelectScreen: React.FC<StationSelectProps> = ({ navigation }) => {
+  const [departStation, setDepartStation] = useState('');
```

Comment:
The empty string here might be a bit confusing and lead to some unexpected behavior. A nicer way to deal with this would be to initialise the `useState` hook as a nullable string with initial value `null`. You can do that like this:

```typescript
useState<string | null>(null);
```